arches_controlled_lists/src/arches_controlled_lists/widgets/ReferenceSelectWidget/components/ReferenceSelectWidgetViewer.vue

```diff
@@ -1,21 +1,11 @@
 <script setup lang="ts">
-import type { ReferenceSelectFetchedOption } from "@/arches_controlled_lists/widgets/types";
+import type { ReferenceSelectValue } from "@/arches_controlled_lists/datatypes/reference-select/types";
 
 defineProps<{
-    value: ReferenceSelectFetchedOption[] | undefined;
+    aliasedNodeData: ReferenceSelectValue;
 }>();
 </script>
 
 <template>
-    <template v-if="value">
-        <span
-            v-for="referenceItem in value"
-            :key="referenceItem.list_item_id"
-        >
-            {{ referenceItem.display_value }}
-        </span>
-    </template>
-    <span v-else>
-        <span>{{ $gettext("None") }}</span>
-    </span>
+    <span>{{ aliasedNodeData.display_value }}</span>
```

Comment:
This is probably okay for now, but it seems likely to me that the viewer should actually render links for each reference here with the URI as the href.  Seems like having querysets transform multiple references into a single display value blocks us from doing so.  is that correct?